### DISCUSSION TITLE
when will the macos release available

### DISCUSSION THREAD

**Discussion by muyiwhy** [kingToolbox/WindTerm#46] [General]
when will the macos release available?
Looked all the features and think this is the exact ssh tool i was looking for

**Comment by kingToolbox** [comment]
According to the version plan, the MacOS version is expected to be released at the end of March or early April. However, according to the current development progress, I think it may be released in advance.

At present, I am more concerned about fixing bugs, improving the support of various protocols, as well as Mosh, x11 of Linux version, ProxyCommand, X/Y/Z modem, memory compression, search sessions, auto-completion and other functions.

**Reply by muyiwhy** [reply]
Looking forward to the MacOS version

**Reply by kingToolbox** [reply]
No problem, I believe it will be released by early April at the latest. :smiley: In addition, I also hope that new M1 version can be released in the second half of this year.

**Comment by kingToolbox** [comment]
@muyiwhy Hello, I am happy to inform you that the MacOS version of WindTerm has been released. Please download and use [WindTerm_2.1.0_Prerelease_6_MacOS_Portable](https://github.com/kingToolbox/WindTerm/releases/tag/2.1-prerelease). The development of `X/Y/Z Modm` and `Sync Input` has delayed the original plan, but I hope it is not too late. :smile: Feedback is welcome if you have any questions. Thank you.

**Reply by muyiwhy** [reply]
Thanks for your reply. I will try on this version.

**Reply by muyiwhy** [reply]
BTW, is there a way to import the session info from SecureCRT now or we have to do it manually now？

**Reply by bestv5** [reply]
> BTW, is there a way to import the session info from SecureCRT now or we have to do it manually now？

  I need this feature  too

**Reply by kingToolbox** [reply]
Sorry, WindTerm only supports automatic import the configuration of OpenSSH and PuTTY, but will not import the configuration of SecureCRT and other commercial softwares. After all, developers of commercial software need to make a living from this.

However, the configuration of both SecureCRT and WindTerm are in very simple plain text format. If you want, you can simply write a script to convert it. :smile:
